.build/Makefile

```diff
@@ -0,0 +1,30 @@
+# set GOOARCH based on ARCH
+ifeq ($(ARCH),x86_64)
```

Comment:
these 3 expressions can be rewritten with `else`:
```
ifeq ($(ARCH),x86_64)
  GOARCH=amd64
else ifeq ($(ARCH),x86)
  GOARCH=386
else
  $(error "ARCH must be set to one of: x86, x86_64")
endif
```

Reply:
Much nicer, thnx. Note as much as it looks weird to not indent the code inside the conditions, I did find I had to not indent the variable setters otherwise I would get an error.